source/frontend/src/features/mapSideBar/acquisition/common/GenerateForm/hooks/useGenerateH0443.ts

```diff
@@ -51,7 +44,7 @@ export const useGenerateH0443 = () => {
         file.acquisitionFileOwners?.filter((x): x is Api_AcquisitionFileOwner => !!x) || [];
       const contactOwner = owners.find(x => x.isPrimaryContact === true);
 
-      const h0443Data: H0443Data = {
+      let h0443Data: H0443Data = {
```

Comment:
nit: I think there is no need to change this from **const* to **let** since you are not re-assigning the whole H0443Data object, you are just assigning some sub-properties of an existing object so **const** can still apply here I think

Reply:
Updated